content/applications/general/payment_acquirers.rst

```diff
@@ -1,12 +1,238 @@
-=================
-Payment Acquirers
-=================
+:show-content:
+
+================================================
+Payment Acquirers (Credit Cards, Online Payment)
+================================================
 
 .. toctree::
    :titlesonly:
 
-   payment_acquirers/payment_acquirers
    payment_acquirers/wire_transfer
    payment_acquirers/authorize
    payment_acquirers/paypal
    payment_acquirers/stripe
+
+Odoo embeds several **payment methods** that allow your customers to pay on their *Customer Portals*
+or your *eCommerce website*. They can pay Sales Orders, invoices, or subscriptions with recurring
+payments with their favorite payment acquirers, including **online payment providers** that accept
+**Credit Cards**.
+
+Having several payment methods increases the chances of getting paid in time, or even immediately,
+as you make it more convenient for your customers to pay with the payment method they prefer and
+trust.
+
+.. image:: payment_acquirers/online-payment.png
+   :align: center
+   :alt: Pay online in the customer portal and select which payment acquirer to use.
+
+.. important::
+   Odoo does not keep Credit Card numbers or credentials on its servers, nor is it stored on Odoo
+   databases hosted elsewhere. Instead, Odoo apps use a unique reference to the data stored in the
+   payment acquirers' systems, where the information is safely stored. This reference is useless
+   without your credentials for the payment acquirer.
```

Comment:
@AntoineVDV 
It means that even if someone were to retrieve that reference, they won't be able to use it somewhere else and steal money from the credit cards.
Basically, the payment process is safe.

Reply:
Ok, I see what you meant now. For some reason, I thought that by "references" you meant the card number of something.

I still believe that it's not super clear and actually too much detailed. Instead, I'd convert the admonition to a `note` since there is nothing to worry about and say something like:
> Odoo apps delegates handling of sensitive information to the certified payment acquirer so that you don't ever have to worry about PCI-compliance.